specs/MultiProfile.md

```diff
@@ -22,6 +22,12 @@ multiple WebView2s running with separate profiles under a single user data direc
 browser instance at runtime), which means separate cookies, user preference settings, and various
 data storage etc., to help you build a more wonderful experience for your application.
 
+In order to manage cookies through profile, we're adding a get_CookieManager interface into profile.
+Users can use this interface to get the cooke manager, which is shared by all webview2s associated with
+this profile. Currently the cookie manager is got from WebView2 (see ICoreWebView2_2.get_CookieManager),
+as a standalone profile object can not support cooke management APIs. We will make cookie management
```

Comment:
Not sure what this is trying to say. "You have to get the cookie manager from WebView2 because the profile object doesn't support cookie management." So why don't we add cookie management to the profile?

Not sure what the second sentence is trying to say.  "We'll make it independent when we find a way to make it independent."

Reply:
Yes, we will add cookie management to the profile. What I meant was, Ideally, we should be able to get a Cookiemanager object without creating a WebView2, but it is a long term work to separate the cookie manager from WebView2, so currently we only provide a useable interface for users to get the cookie manager for the specified profile, and it calls ICoreWebView2_2.get_CookieManager in the low level. Maybe I should not mention this (about low-level implement) in the API spec?

Reply:
Yes you don't need to mention that it is calling get_CookieManager internally although its fine to say (as you do say elsewhere) that the CoreWebView2.Profile.CookieManager is the same as CoreWebView2.CookieManager.